cartridges/int_stickyio_sfra/cartridge/client/default/js/stickyio/stickyioSubscriptionManagement.js

```diff
@@ -71,6 +76,43 @@ $('body').on('click', '.stickyioconfirmbutton', function () {
     });
 });
 
+$('body').on('click', '.stickyAddressShow', function () {
+	$('.stickyioSubscriptionResponse').removeClass('stickyiosubmansuccess','stickyiosubmanerror').text('').hide();
+});
+$('body').on('click', '.stickyCreditCardShow', function () {
+	$('.stickyioPaymentResponse').removeClass('stickyiosubmansuccess','stickyiosubmanerror').text('').hide();
+});
+
+$('.stickyAddress-form').on('submit', function (e) {
+	e.preventDefault();
+	
+    $.ajax({
+        url: $(this).attr('action'),
+        method: 'POST',
+        data     : $(this).serialize(),
+        dataType: 'json',
+        success: function (data) {
+        	
```

Comment:
Is it possible to remove this blank line and correct the indentation below?

Reply:
Fixed